src/main/java/com/commands/PutMineCommand.java

```diff
@@ -11,8 +11,10 @@ public PutMineCommand () {
 	}
 	
 	public void execute(Vehicle vehicle) {
-		// TODO: set Positions
-		
+		// TODO: set Positions}

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)